src/shared/utilities/tsUtils.ts

```diff
@@ -43,7 +50,7 @@ export function isNonNullable<T>(obj: T): obj is NonNullable<T> {
 }
 
 export function isKeyOf<T>(key: PropertyKey, obj: T): key is keyof T {
-    return key in obj
+    return key in (obj as any)
```

Comment:
🤷

Reply:
Probably have to narrow `T` with `T extends Record<PropertyKey, any>`. Not sure why we have linting on for `object` as that'd be the most accurate here.